java-to-zod-gradle-plugin/src/test/java/sh/ivan/zod/GenerateZodSchemasTest.java

```diff
@@ -0,0 +1,84 @@
+package sh.ivan.zod;
+
+import org.gradle.testkit.runner.BuildResult;
+import org.gradle.testkit.runner.GradleRunner;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.io.TempDir;
+
+import java.io.File;
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.nio.file.StandardCopyOption;
+import java.util.Arrays;
+
+import static org.gradle.testkit.runner.TaskOutcome.SUCCESS;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+public class GenerateZodSchemasTest {
```

Comment:
I personally think we should delete this test. It looks way too reliant on internal details of how Gradle plugins work. I'm more interested in asserting that the can be used from a regular old Gradle project, and generate a file that meets expectations. This is what we test in the Maven plugin. Thoughts?